operator-metadata/automation/tests/resources/new.format.internal.pull.specs.test.bundle.clusterserviceversion.yaml

```diff
@@ -361,7 +361,7 @@ metadata:
     capabilities: Deep Insights
     categories: Streaming & Messaging
     certified: 'false'
-    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/amq-streams-strimzi-rhel9-operator:2.5.0-1
+    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/amq-streams-strimzi-rhel8-operator:2.5.0-1
```

Comment:
curious, I know it is only test data, but why switch back to rhel8?

Reply:
The other example test data we have from CPaaS e.g. the contents contained within files like `CONTAINER_BUILDS_BRIDGE_BUILD_INFO_JSON` reference older images that have rhel8 pull specs. Since the test relies this CPaaS test data I needed to align either the CPaaS test data to the newer rhel9 pull specs or the internal CSV pull specs to older rhel8 pull specs, I chose the latter

Reply:
Okay